qa/tasks/radosgw_admin_rest.py

```diff
@@ -130,8 +133,11 @@ def task(ctx, config):
         config = dict.fromkeys(config)
     clients = config.keys()
 
+    if config.get('endpoint'):
+        client = config.get('endpoint')
```

Comment:
i think you can already accomplish this by providing a list of one client in the task's config, ie
```yaml
- radosgw-admin-rest: [c2.client.0]
```
the `if isinstance(config, list):` block will use that list instead of `all_clients`, so `clients[0]` below will select the desired endpoint